test/e2e/query_test.go

```diff
@@ -197,3 +204,18 @@ func queryPrometheus(ctx context.Context, ustr string, ts time.Time, q string, d
 	}
 	return m.Data.Result, nil
 }
+
+func defaultPromConfig(name string, replicas int) string {
```

Comment:
What is this? Im abit confused as to why we have this in the code? Is it copypasta?